spi/src/main/java/com/amazon/opendistroforelasticsearch/jobscheduler/spi/ScheduledJobParameter.java

```diff
@@ -55,4 +55,9 @@
     default Long getLockDurationSeconds() {
         return null;
     }
+
+    /**
+     * @return job execution jitter, 0 <= jitter <= 1
```

Comment:
better to add some comment to explain how jitter works here

Reply:
sure, will add